.gitignore

```diff
@@ -28,3 +28,5 @@ strace*.txt
 .cache
 wheels
 *.kate-swp
+# From pip install --user -e . # (development mode):
```

Comment:
Uhh...

Did you accidentally commit this file? (I actually need `sdbus.egg-info` to stay out of gitignore)

Reply:
I thought it would not hurt and that you'd see it in the diff of the PR (sorry for not mentioning it in the text)